### DISCUSSION TITLE
Support From Erlang Foundation

### DISCUSSION THREAD

**Discussion by rupurt** [ergo-services/ergo#141] [General]
Have you considered apply for support from the [Erlang Foundation](https://erlef.org/)? I feel like they would love to broaden the user base of OTP :smile:

**Comment by halturin** [comment]
Not sure my project meets their requirements :)

PS: thanks for your sponsorship. I appreciate it a lot 👍. BTW, as a sponsor, you will get access to the Cloud Overlay Network  https://github.com/sponsors/ergo-services (in a week or two)

**Reply by rupurt** [reply]
Haha, I think we can make them come around to the idea :smirk: 

My pleasure on the sponsorship! Hopefully I will be able to add more this year. It's hard to describe how big of a deal OTP in golang is. There are so many people that won't even consider Elixir/Erlang because it doesn't have a static type system.
